client/scss/utilities/_root-scope-classes.scss

```diff
@@ -292,3 +292,10 @@
 .absolute {
   position: absolute;
 }
+
+// This is for when we receive content from a CMS,
+// making it easier to style the first node
+.first-para-bold p:first-of-type {
+  @include font('HNM3');
+  font-weight: bold;
```

Comment:
Probably don't want to be applying bold to the HNM3 font, but this has made me think we should tweak the way we handle the fonts slightly. Will talk it over with you.